app/controllers/bookmarks_controller.rb

```diff
@@ -13,6 +13,23 @@ class BookmarksController < CatalogController
       }
   end
 
+  def index
```

Comment:
Assuming this was copied from somewhere and modfiied, can you add a comment linking to where it was copied from and describing the modification? This sort of thing can be helpful during upgrades.